src/webob/acceptparse.py

```diff
@@ -255,11 +294,382 @@ class AcceptEncoding(Accept):
     """
 
 
-class AcceptLanguage(Accept):
+class AcceptLanguage(object):
     """
-    Represents an ``Accept-Language`` header.
+    Represent an ``Accept-Language`` header.
+
+    Base class for :class:`AcceptLanguageValidHeader`,
+    :class:`AcceptLanguageNoHeader`, and :class:`AcceptLanguageInvalidHeader`.
     """
-    def _match(self, mask, item):
+
+    @classmethod
+    def python_value_to_header_str(cls, value):
```

Comment:
Add an underscore to the name. This way it is markedly non-public and makes it simpler to see so when reading the code.